Formula/wxwidgets@3.0.rb

```diff
@@ -10,21 +10,13 @@ class WxmacAT30 < Formula
     regex(/^v?(\d+\.\d*[02468](?:\.\d+)*)$/i)
   end
 
-  bottle do
-    sha256 cellar: :any,                 arm64_big_sur: "781881c92fcaadcfea66e7d33a36ae1d9b55d26839738f44248f39d187f46a3c"
-    sha256 cellar: :any,                 big_sur:       "42385ebead8045f2f4b1ac40f13f15c18b84b78ffa7110d69ebf8a798cd29dc0"
-    sha256 cellar: :any,                 catalina:      "d3d18b83d84b79735bda4497ebd92f763ed7cf4200c8ce8d651070ccd1b2719d"
-    sha256 cellar: :any,                 mojave:        "319f6ca3509265ace16fae3aaab777ae932130b741ba51d99702c0391da7833f"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "7c20b17371eaf47ea6da7f6754d35d539f335bec8ce11f3d0c6c41a113ffbbe8"
-  end
-
   depends_on "jpeg"
   depends_on "libpng"
   depends_on "libtiff"
 
   on_linux do
     depends_on "pkg-config" => :build
-    depends_on "gtk+"
+    depends_on "gtk+3"
```

Comment:
I think this one still needs `gtk+`. This was actually the big draw of 3.1 -- compatibility with a not-super-old version of GTK

Reply:
Based on Linux distros and repositories, I thought it would work with GTK+3 unless there are some backports necessary.

For example, the main homepage links to CodeLite for Debian/Ubuntu at https://wiki.codelite.org/pmwiki.php/Main/WxWidgets30Binaries#toc2 which says:
> Also starting with wx3.0.5 I've only built packages against GTK+3.

Haven't tested it though, so I don't know for sure.

Reply:
Hmmm, maybe I am misinformed. I guess it's worth a try.